packages/babel-generator/test/fixtures/typescript/types-with-comments-retainLines/output.js

```diff
@@ -6,8 +6,8 @@ C /* 2 */[
 
 type T2 = U.
 /* 1 */
-C /* 2 */[];
-
+C /* 2 */[]
+/* 3 */;
```

Comment:
`/* 3 */` was not printed, this output was wrong!

It should be printed inside `[]`, but we print `[]` as a single token. I'll open another PR for that.